### PR TITLE
Tariff id in site area

### PR DESCRIPTION
**Pull request by rmangin-fr**

### CODE REVIEW

**Review by ClaudeROSSI — APPROVED** [review]
ok

File: src/app/pages/organization/site-areas/site-area/site-area.component.html [hunk 1]
```diff
@@ -167,6 +167,26 @@ <h5 class="card-title">
...
+                  [formControl]="tariffID" (input)="emptyStringToNull(tariffID)"/>
+                <mat-error *ngIf="tariffID?.errors?.required">{{"general.mandatory_field" | translate}}</mat-error>
+                <mat-error *ngIf="tariffID?.errors?.maxlength">
+                  <div [translate]="'general.error_max_length'" [translateParams]="{length: 36}"></div>
+                </mat-error>
+              </mat-form-field>
+            </div>
+          </div>
+        </div>
+      </mat-tab>
```
Comment (ClaudeROSSI) [inline src/app/pages/organization/site-areas/site-area/site-area.component.html:189]: As far as I understood, Serge would like to split this in two components (one component per tabs).
Something similar to what has been done for sites (with site/main & site/site-ocpi).
@OliveGerste : did you discuss that point with Remi?

image

**Review by ClaudeROSSI — CHANGES_REQUESTED** [review]
please heck my comment

**Review by ClaudeROSSI — CHANGES_REQUESTED** [review]
Serge wants a component per tabs!

File: src/app/pages/organization/site-areas/site-area/site-area.component.html (hunk 1 above)
Comment (OliveGerste) [reply src/app/pages/organization/site-areas/site-area/site-area.component.html:189]: I thought about this as well, but then it seemed more like a refactoring to me. 
@LucasBrazi06 Should this be done in this PR or do we create a separate issue for this?

Comment (LucasBrazi06) [reply src/app/pages/organization/site-areas/site-area/site-area.component.html:189]: > I thought about this as well, but then it seemed more like a refactoring to me. @LucasBrazi06 Should this be done in this PR or do we create a separate issue for this?

To be done in this PR.
Thanks!

**Comment by LucasBrazi06** [comment]
@OliveGerste As replied earlier, split the view.
Thanks!

**Comment by OliveGerste** [comment]
> @OliveGerste As replied earlier, split the view. Thanks!

All right. Should I only add the ne component as a new view or do you want me to also split the old component into main and tab component.

**Comment by LucasBrazi06** [comment]
> > @OliveGerste As replied earlier, split the view. Thanks!
> 
> All right. Should I only add the new component as a new view or do you want me to also split the old component into main and tab component like it is done in site already?

Like the Site, thanks!!!

**Review by LucasBrazi06 — APPROVED** [review]

**Pull request merged**